src/app/ngrx/actions/castle.action.creator.ts

```diff
@@ -1,11 +1,12 @@
-import { ActionTypes, ActionWithPayload } from './action.types';
+import { Action } from '@ngrx/store';
+import { ActionTypes } from './action.types';
 import { FilterModel, EpisodeModel, PodcastModel, MetricsType } from '../model';
 
 export interface CastleFilterPayload {
   filter: FilterModel;
 }
 
-export class CastleFilterAction implements ActionWithPayload<CastleFilterPayload> {
```

Comment:
`ActionWithPayload` came from the [migration guide](https://github.com/ngrx/platform/blob/master/MIGRATION.md#action-interface) and it did not pan out. It was causing TypeScript to complain about my ActionReducerMap, which was a red herring on that problem with the reducers not getting called last week.